islandora_compound_object.module

```diff
@@ -450,18 +470,15 @@ EOQ;
     uasort($results, $sort);
 
     foreach ($results as $result) {
-      $object = islandora_object_load($result['object']['value']);
-      if (islandora_object_access(ISLANDORA_VIEW_OBJECTS, $object)) {
```

Comment:
I think it almost needs to exist for legacy purposes because there could be an implementation out there that is employing its own custom access controls. We could likely do something like this:

- Change the code to have the object_access off by default with a `variable_get` and reap all the sweet sweet non-loading benefits.
- Have an update hook that sets the variable mentioned previously to make object_access be on to preserve the previously set behavior